src/main/java/Duke/commands/AddDeadlineCommand.java

```diff
@@ -0,0 +1,44 @@
+package Duke.commands;
```

Comment:
You may want to consider changing names representing packages to all lowercase, to conform to the coding standard.